firefox-ios/Ecosia/L10N/en.lproj/Ecosia.strings

```diff
@@ -214,27 +213,22 @@
 "We’ve added 1Password to Ecosia to help you to manage your passwords securely." = "We’ve added 1Password to Ecosia to help you to manage your passwords securely.";
 "Got it" = "Got it";
 
-// APN Consent
-"Keep up with Ecosia" = "Keep up with Ecosia";
-"Turn on push notifications" = "Turn on push notifications";
-"Discover the trees we plant and the impact they’re having" = "Discover the trees we plant and the impact they’re having";
-"Get tips on how to help build a greener future" = "Get tips on how to help build a greener future";
-"Receive updates on our tree-planting projects around the world" = "Receive updates on our tree-planting projects around the world";
-"Get tips on how to be climate active every day" = "Get tips on how to be climate active every day";
-"Allow push notifications" = "Allow push notifications";
-"Not now" = "Not now";
-
 // Newsletter
 "Be the first to know" = "Be the first to know";
 "Subscribe to our monthly newsletter for updates on your climate impact." = "Subscribe to our monthly newsletter for updates on your climate impact.";
 "Sign up" = "Sign up";
 
 // Default Browser
+"Open settings" = "Open settings";
```

Comment:
This is the only change that would currently affect production, since we now use `Open settings` instead of `Go to settings` (removed from the line above).

Still see no issues as the key is the same and all other languages will be for now using the old translation.